nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -43,24 +43,52 @@ class PostTrainingQuantizationParameters(AlgorithmParameters):
     """
 
     def __init__(self,
+                 number_samples: int = 300,
                  preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
-                 weight_bits: int = 8,
-                 weight_granularity: Granularity = Granularity.PERCHANNEL,
-                 activation_bits: int = 8,
-                 activation_granularity: Granularity = Granularity.PERTENSOR,
+                 weight_bits: Optional[int] = None,
+                 weight_granularity: Optional[Granularity] = None,
+                 weight_signedness_to_force: Optional[bool] = None,
+                 activation_bits: Optional[int] = None,
+                 activation_granularity: Optional[Granularity] = None,
+                 activation_signedness_to_force: Optional[bool] = None,
+                 target_device: TargetDevice = TargetDevice.ANY,
                  range_type: RangeType = RangeType.MEAN_MINMAX,
-                 number_samples: int = 300,
-                 target_device: HWConfigType = HWConfigType.CPU,
                  quantize_outputs: bool = False,
                  ignored_scopes: Optional[List[str]] = None,
                  fast_bias_correction: bool = True,
                  ):
+        """
+        :param number_samples: Number of samples for the statistics collection.
+        :param preset: Preset parameter for Quantization.
+        Defines the mode: symmetric or asymmetric of the activation quantizers.
```

Comment:
Could you format the docstring according to the Python style guide:
```suggestion
            Defines the mode: symmetric or asymmetric of the activation quantizers.
```

Reply:
Done